finality-provider/config/evm.go

```diff
@@ -10,18 +10,21 @@ const (
 )
 
 type EVMConfig struct {
-	RPCAddr string `long:"rpc-address" description:"address of the rpc server to connect to"`
+	RPCL1Addr                      string `long:"rpc-address" description:"address of the L2rpc server to connect to"`
```

Comment:
`L2rpc` is not one word. should be `L2 RPC`

Reply:
change the name